src/modules/gpu.c

```diff
@@ -25,11 +26,14 @@ static void printGPUResult(FFinstance* instance, uint8_t index, FFcache* cache,
     if(gpu->coreCount != FF_GPU_CORE_COUNT_UNSET)
         ffStrbufAppendF(&output, " (%d)", gpu->coreCount);
 
+    if(!isnan(gpu->temperature)) //FF_GPU_TEMP_UNSET
```

Comment:
On my system, `isnan` requires linking `libm`, i don't think thats worth over simple `gpu->temperature != gpu->temperature`.

Same goes for the other `isnan` calls.